### DISCUSSION TITLE
how do skywalking 8.9 ui shows Global Brief

### DISCUSSION THREAD

**Discussion by zj452008181** [apache/skywalking#8771] [Q&A] [answered]
skywalking 6.6 has the Global Brief template,it can shows the service and databases total numbers,it's usefule for me,but in skywalking 8.x,you remove it,how can i set it in skywalking 8.9
![image](https://user-images.githubusercontent.com/26154743/160958476-cb648e69-3782-4932-9bcf-7950b340fd0b.png)

**Comment by wu-sheng** [comment, marked as answer]
No, it is removed permnantly.
